coder_sniffer/DrupalPractice/Sniffs/Objects/GlobalClassSniff.php

```diff
@@ -112,4 +169,53 @@ public function process(File $phpcsFile, $stackPtr)
     }//end process()
 
 
+    /**
+     * Retrieve the fully qualified name of the given classname.
+     *
+     * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
+     * @param string                      $className The classname for which to retrieve the FQN.
+     *
+     * @return string
+     */
+    protected function getFullyQualifiedName(File $phpcsFile, $className)
```

Comment:
this is a bit sad that we need to implement this ourselves. PHPCS core should have a helper for this. I don't see a better way currently, so I think this ok.

Reply:
Yes, it would be nice to have an implementation for this upstream, for now this seems to be ok. If we ever need this information again then we might consider moving this to a coder util class or something :slightly_smiling_face: